snoop/wsgi.py

```diff
@@ -9,5 +9,6 @@
 
 from django.conf import settings
 from . import defaultsettings
-if not settings.DEBUG and settings.SECRET_KEY == defaultsettings.SECRET_KEY:
-    raise RuntimeError("Please change the default SECRET_KEY setting")
+if settings.SECRET_KEY == defaultsettings._default_secret_key:
```

Comment:
Why split in 2 branches?

Reply:
To avoid a long line, so that it's easier to read.

Reply:
To me it was perfectly readable as it was. Adding more branches increases complexity, while consolidating them reduces it.

Reply:
Ok, how about now?

Reply:
OK